### PR TITLE
Update promotion.md

### PR DESCRIPTION
**Pull request by jt15s**
Trying (again) to fix the page rendering issue.

### CODE REVIEW

File: content/community/getting-involved/promotion.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
----
++++
 type = "article"
 title: "Promotion"
 date: 2021-04-14T15:16:45+10:00
 tags = []
----
++++
 Promotion is a crucial part of ensuring Haiku continues to grow. Without spreading the word, we cannot gain new users and volunteers.
```
Comment (humdingerb) [inline content/community/getting-involved/promotion.md:7]: I'd add an empty line after the `+++`

**Review by humdingerb — COMMENTED** [review]
github doesn't allow commenting lines that were not changed.

You still need to add 4 or 5 spaces for line-breaks and an empty line before all bullet lists.

**Comment by jt15s** [comment]
@humdingerb Is this alright?

File: content/community/getting-involved/promotion.md [hunk 2]
```diff
@@ -30,46 +31,45 @@ To help spread the word and let more people know about Haiku, and helping to imp
...
+scott_puopolo18 is working on a set of interview questions for our developers to give the public a glimpse into the world of Haiku development, as well as shining a light on the challenges and opportunities in an open-source project. The predetermined set of questions means that we don't have to write new questions for each developer. To save time, we can simply send them off and the developer can write their response.     
 We are also planning to distribute more content via Haiku's official social media platforms. Approval has been given for a Haiku "brand-account" to be set up on YouTube. Combined with the Haiku account on Odysee (another platform similar to YouTube), these two accounts will be an outlet for video-based content from the promotion team. Other more text-based social media accounts on platforms such as Twitter will remain as they are, retweeting news from the Project as well as other relevant content.
```
Comment (humdingerb) [inline content/community/getting-involved/promotion.md:73]: A couple of spaces after "content".

**Review by humdingerb — COMMENTED** [review]
Just that one small thing, then we're golden. If the initial problem with the page is now fixed...

**Comment by jt15s** [comment]
@humdingerb Should be all good now!

**Pull request merged**